packages/components-vue/vite.config.ts

```diff
@@ -16,6 +17,14 @@ export default defineConfig({
   plugins: [
     vue(),
     vueJsx(),
+    createHtmlPlugin({
```

Comment:
Is this even needed? I think this config is used to build the package, not the app.